docs/UserGuide.md

```diff
@@ -151,82 +153,170 @@ LibTask's data is saved in the hard disk automatically after any command that ch
 
 ### 3.2. Patron Features
 
+This section describes features users can use to interact with LibTask's patron list.
+
 --------------------------------------------------------------------------------------------------------------------
 #### 3.2.1. Adding a patron: `patron add`
 Adds a patron to LibTask's patron list.
 
 Format: `patron add n/NAME s/ID p/PHONE e/EMAIL [t/TAG]…​`
 
-Examples:
-* `patron add n/John s/A02128282A p/93231222 e/e03482@u.nus.edu t/student`
-* `patron add n/Alice s/S01823283S p/90123212 e/profA@u.nus.edu`
+**Notes about the add command:**
+
+* ID must be 9 characters in length. First character of ID should be 'A' and last character of ID should be an alphabet.
+* You cannot add a patron with the same NAME, EMAIL or ID as another existing patron in the patron list.
+
+**Example**
```

Comment:
I think you might want to add the following restrictions:

You can add the email length restriction.
You can also add the name length restrictions.
🤔